android/build.gradle

```diff
@@ -117,8 +118,6 @@ android {
                 srcDir 'src'
             }
         }
-
-        plus1.res.srcDirs = ['res-plus1']
```

Comment:
seems like this will cause issues.

Reply:
oh nvm, that "FrostWire Plus" string is the only thing, good call.

Reply:
Awesome commit!

One favor.

Please prefix your commits messages with "[android] " if you're changing files inside android/
"[common] " for common
and
"[desktop] " if you ever do anything for desktop.

Reply:
Ok, I'll try to remember that for the future.